src/hooks/useClientLocation.js

```diff
@@ -1,11 +1,11 @@
+// @ts-nocheck
```

Comment:
Can you add a screenshot and description of the error on this place, please?

Reply:
![image](https://user-images.githubusercontent.com/61360270/142058729-03756238-00d4-4d51-8529-5b17be2a261c.png)

Reply:
We should avoid this `isEqual` method, or at least verify if there are not better options.

@Katsu08 please add a ticket to investigate this one